### ISSUE TITLE
Not all information printed at standard output ends up in log file

### ISSUE THREAD

**Issue by wawrzek** [corda/corda#305]
I noticed that some of the information printed by Corda at standard output are not visible in log file. Not sure what's exactly the difference, but today I noticed that the line informing about loaded plugins/CorDapps are not in logs. Please find this example:
`Loaded plugins                          : com.example.plugin.ExamplePlugin, net.corda.cordappb.plugin.CordappBPlugin, net.corda.cordappd.plugin.CordappDPlugin, net.corda.cordappc.plugin.CordappCPlugin` 
is not visible in logs.

**Comment by wawrzek** [comment]
Example of initial information not in log. Of course we don't need Corda ASCII art in logs ;). But information lines about Loaded plugins,  Database connection and Node listening address, network services are very useful.

```
Mar 13 12:13:47 pentagon-r3 java[39352]:    ______               __
Mar 13 12:13:47 pentagon-r3 java[39352]:   / ____/     _________/ /___ _
Mar 13 12:13:47 pentagon-r3 java[39352]:  / /     __  / ___/ __  / __ `/         It's not who you know, it's who you know
Mar 13 12:13:47 pentagon-r3 java[39352]: / /___  /_/ / /  / /_/ / /_/ /          knows what you know you know.
Mar 13 12:13:47 pentagon-r3 java[39352]: \____/     /_/   \__,_/\__,_/
Mar 13 12:13:47 pentagon-r3 java[39352]: --- MILESTONE 9 -------------------------------------------------------------------
Mar 13 12:13:47 pentagon-r3 java[39352]: 📚  New! Training now available worldwide, see https://corda.net/corda-training/
Mar 13 12:13:49 pentagon-r3 java[39352]: Logs can be found in                    : /opt/corda/logs
Mar 13 12:13:53 pentagon-r3 java[39352]: Database connection url is              : jdbc:h2:tcp://10.10.10.8:11000/node
Mar 13 12:14:00 pentagon-r3 java[39352]: Node listening on address               : pentagon-r3.corda.r3cev.com:10002
Mar 13 12:14:02 pentagon-r3 java[39352]: Providing network services              :
Mar 13 12:14:02 pentagon-r3 java[39352]: Loaded plugins                          : com.example.plugin.ExamplePlugin
Mar 13 12:14:09 pentagon-r3 java[39352]: Node started up and registered in 23.77 sec
```

**Comment by kchalkias** [comment]
@wawrzek by default we set `renderBasicInfoToConsole = true`, unless is set to false as a command line argument. This affects any part of the code that calls `printBasicNodeInfo`, I am enlisting its usages:
```
printBasicNodeInfo("Node started up and registered in $elapsed sec")
printBasicNodeInfo("Logs can be found in", System.getProperty("log-path"))
printBasicNodeInfo("Loaded plugins", plugins.joinToString())
printBasicNodeInfo("Database connection url is", "jdbc:h2:$url/node")
printBasicNodeInfo("Node shutting down ...")
printBasicNodeInfo("Node listening on address", myHostPort.toString())
printBasicNodeInfo("Starting as webserver: ${config.webAddress}")
```
Should I change default to false?

**Comment by mikehearn** [comment]
The list of plugins should be printed to the logs - I suspect this is a bad interaction between logging and newlines or something like that. The defaults shouldn't need changing, but we can probably just introduce a few new functions to better distinguish what should be printed where.

**Comment by kchalkias** [comment]
Actually, our current policy states that anything calling printBasicNodeInfo, won't be printed to the log file unless `renderBasicInfoToConsole = false`. As this is how we currently distinguish what should be printed in the logs or not, a quick solution would be to just change `printBasicNodeInfo("Loaded plugins", plugins.joinToString())` to `log.info("Loaded plugins", plugins.joinToString())`.

**Comment by mikehearn** [comment]
Oh, I see. We want the stuff to continue to be printed to the console as well though. This should be a super easy fix.

**Comment by kchalkias** [comment]
Is it only the list of plugins or others as well? (from the aforementioned list).

**Comment by wawrzek** [comment]
Probably, everything (except `printBasicNodeInfo("Logs can be found in", System.getProperty("log-path"))`. It's a bit redundant to write in logs, where you can find theim ;). But if that easier we can have it in logs as well.

**Comment by kchalkias** [comment]
See fix here: https://github.com/corda/corda/pull/356

**Comment by mikehearn** [comment]
Whilst we're at it, can we print the node's legal name to the screen and log file as well?

https://discourse.corda.net/t/cant-connect-to-bob-cuz-hes-living-somewhere-else/920

**Comment by kchalkias** [comment]
fixed here: [#356](https://github.com/corda/corda/pull/356)
